ORB_SLAM3_RGBD_ROS2:
  ros__parameters:
    robot_base_frame: base_footprint
    global_frame: map
    odom_frame: odom
    robot_x: 0.0
    robot_y: 0.0
    visualization: true
    ros_visualization: false